hack/build-restic.sh

```diff
@@ -1,56 +0,0 @@
-#!/bin/bash
```

Comment:
The change looks good, but please don't delete the `build-restic.sh` file.
Tiltfile also references the `build-restic.sh` file.

Reply:
alright.